22_pipeslab/pipeslab.txt

```diff
@@ -2,20 +2,44 @@ For each of the following, type a single command line that performs:
 
   1. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner.
   Answer: 
+	ls -l|grep $USER| cut -c 4|grep x|wc -w
+	[Q: Based on the commands we know by now is there a shorter way to do 	this? With less pipes?] 
 
```

Comment:
You don't need the first `grep`, as the question is not restricted to YOUR files. And one nice shortcut is to use `grep -c` (which just prints a count). Therefore you can use: `ls -l | cut -c 4 | grep -c x`